src/settings.ts

```diff
@@ -185,9 +185,13 @@ export async function parseSettings(
   );
 
   await check("afterReset", async (rawAfterReset: unknown) => {
-    const afterResetArray = Array.isArray(rawAfterReset)
-      ? rawAfterReset
-      : [rawAfterReset];
+    let afterResetArray;
+    afterResetArray = !rawAfterReset ? [] : [rawAfterReset];
+
+    if (Array.isArray(rawAfterReset)) {
+      afterResetArray = rawAfterReset;
+    }
+
```

Comment:
I think a neater solution would be to put `if (!rawAfterReset) {return;}` at the top of the function. Always prefer const over let if possible.

Reply:
Definitely a neater solution. thx